src/vs/workbench/contrib/chat/common/chatService.ts

```diff
@@ -313,12 +313,9 @@ export interface IChatTerminalToolInvocationData {
 	alternativeRecommendation?: string;
 	language: string;
 	terminalToolSessionId?: string;
-	terminalCommandUri?: UriComponents;
+	/** The predefined command ID that will be used for this terminal command */
+	terminalCommandId?: string;
 	autoApproveInfo?: IMarkdownString;
-	output?: {
-		html: string;
-		truncated?: boolean;
-	};
```

Comment:
Does deserializations still restore the output after reload if the terminal's gone? We want it so this works:

1. Run ls
2. Expand in chat, works
3. Collapse in chat
4. Kill the terminal
5. Expand in chat, works
6. Collapse in chat
7. Reload window
8. Expand in chat, works (this is what I'm worried about)

Reply:
Yes please, when output is ready we should serialize it in addition to the command URI so that we can toggle even when the terminal's long gone. Otherwise looking back at history will omit all the output